data/categories.txt

```diff
@@ -68,7 +68,7 @@ sk:Jedlo
 @transport
 en:Transport
 ru:Транспорт
-ar:النقل
+ar:مواصلات|نقل
```

Comment:
Ok, 
I will add the '^' sign front of the preferable translation.
Thanks
But why sometimes I see this sign (^) in translations that have one synonym ???